.gitignore

```diff
@@ -7,3 +7,5 @@ spec/target
 **/fuzz/corpus/
 **/fuzz/target/
 **/fuzz/artifacts/
+/crates/cli/cli-tests/target/
+/crates/cli/tests/testfiles/outdir
```

Comment:
Just test files for `cli-tests`.

Reply:
Please put docs for those things in place because questions such as these are going to arise sooner or later. Probably the best would be a `README` for the `wasmi_wasi` or other WASI specific `wasmi` crates.

Reply:
Sure. Will do that